accepted/00102-upgrade-to-salt-3008.md

```diff
@@ -0,0 +1,202 @@
+
+#### Runtime:
+- `python-networkx` + its dependencies: `python-matplotlib`, `python-pandas`, `python-scipy`, `python-FontTools`)
+- `python-tornado6`
+
+#### BuildTime:
+- `python-networkx` -> `python-pydot`, `python-pygraphviz`, `graphviz`
+- `python-meson-python` -> `meson`, `patchelf`
+- `python-pandas` -> `python-versioneer-toml`
+- `python-scipy` -> `python-pythran`
+
+A testing build project can be found [here](https://build.opensuse.org/project/show/home:PSuarezHernandez:branches:systemsmanagement:saltstack:products:next), based on the current upstream Salt `master` branch (target for 3008 release).
+
+## Maintaining Salt
+
+Given the irruption of Salt Extensions, we need to rethink the current workflow we use for maintaining Salt (and now also some of its extensions). In this regard, we want to improve our workflow, to make it simpler and adapted to the new Salt Extensions scenario and aligned with SUSE & OBS proposed workflows.
+
+As this section has itself enough significance, it will be covered in a dedicated RFC around "New SCM workflow for Salt", so discussion around this topic can happen separetely to the present RFC.
+
+# Drawbacks
+[drawbacks]: #drawbacks
+
+The current Salt version used in Uyuni and SUSE Manager is `3006.0` LTS (together with SUSE patches). The EOL for 3006 LTS is currently set by upstream to January 31st 2026.
+
+We should move the Salt version to 3008.x (LTS) at some point before 3006.x is EOL, in order to be covered by Salt upstream support.
+
+If we don't upgrade eventually to the next supported LTS version of Salt, then we would be in our own having to maintain and backport potential security issues detected by Salt into this upstream unsupported version. We want to avoid this situation as much as we can.
+
+Our ultimately goal is to have the Salt version used in Uyuni and SUSE Manager always aligned with the latest upstream supported LTS version of Salt.
+
+# Alternatives
+[alternatives]: #alternatives
+
+Since Salt 3008.x release has been delayed and it is not yet fully clear when it would be, there are some risks that we cannot proceed with the upgrade to Salt 3008.x within some important deadlines when it comes to SUSE Manager and SLE.
+
+1. Stick to the current Salt 3006.0 forever -> no go.
+2. Stick to Salt 3006.0 until 3008 is released -> this will be the way to go if 3008.x is not released on time.
+
+**IMPORTANT: Since we want to drop the Salt 3006.0 from the Basesystem and Server Application modules for SLE 15 SP7 (based on Python 3.6), in favor of a new Salt based on Python 3.11, even if 3008 is delayed we should NOT compromise the SLE 15 SP7 feature cutoff deadline to implement the new "SLE-Module-Salt" module even if it still provides 3006.0 version**
+
+The alternative plan:
```

Comment:
I think we should make clear that this is the plan we will be moving, since salt 3008 should not be on time.